src/core/symbology/qgssymbollayerutils.cpp

```diff
@@ -1296,7 +1296,8 @@ QgsSymbol *QgsSymbolLayerUtils::loadSymbol( const QDomElement &element, const Qg
       }
       else
       {
-        if ( QgsSymbolLayer *layer = loadSymbolLayer( e, context ) )
+        std::unique_ptr< QgsSymbolLayer > layer = loadSymbolLayer( e, context );
+        if ( layer.get() )
```

Comment:
```suggestion
        if ( layer )
```

Reply:
Actually that's a different method which was returning a raw pointer, I've updated that one now too

Reply:
Not sure to understand your comment here. I was just proposing to test the non-null pointer without calling get(), which is useless, no?

Reply:
Ok, fixed